start-script.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# This is the entry script for the docker container. It is called from ta3_search, and is responsible for initializing apache, mysql and grpc-relay services.
```

Comment:
This line is a little bit too long. Usually it is a good idea to have the line length fit in github's default code review box. For example, we can put ", and is responsible for ..." on the second line.